### PR TITLE
fix the wal comment

### PR DESCRIPTION
**Pull request by neuyilan**
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/wal/utils/WALFileUtils.java [hunk 1]
```diff
@@ -105,15 +105,15 @@ public static void ascSortByVersionId(File[] walFiles) {
...
    * Find index of the file which probably contains target insert plan. <br>
-   * Given wal files [ _0-0-1.wal, _1-5-0.wal, _2-5-1.wal, _3-12-1.wal, _4-12-1.wal ], details as
+   * Given wal files [ _0-0-1.wal, _1-5-0.wal, _2-5-1.wal, _3-12-0.wal, _4-12-1.wal ], details as
    * below: <br>
    * _0-0-1.wal: 1, 2, 3, -1, -1, 4, 5, -1 <br>
    * _1-5-0.wal: -1, -1, -1, -1 <br>
-   * _2-5-1.wal: 6, 7, 8, 9, -1, -1, -1, 10, 11, -1, 12, 12 <br>
-   * _3-12-1.wal: 12, 12, 12, 12, 12 <br>
+   * _2-5-1.wal: 6, 7, 8, 9, -1, -1, -1, 10, 11, -1, 12 <br>
+   * _3-12-0.wal: -1, -1, -1, -1, -1 <br>
```
Comment (HeimingZ) [inline server/src/main/java/org/apache/iotdb/db/wal/utils/WALFileUtils.java:113]: These comments don't need to be changed.

**Review by HeimingZ — COMMENTED** [review]
The return values in the comment are incorrect, thanks for fixing it. However, the format of given files are changed incorrectly, because the entries sharing the same seach index should consecutively exist in several continuous files.

File: server/src/main/java/org/apache/iotdb/db/wal/utils/WALFileUtils.java (hunk 1 above)
Comment (neuyilan) [reply server/src/main/java/org/apache/iotdb/db/wal/utils/WALFileUtils.java]: Thanks for pointing out!

**Review by HeimingZ — APPROVED** [review]
LGTM

**Pull request merged**